software/src/modules/ac011k/ac011k.cpp

```diff
@@ -1284,11 +1284,41 @@ void AC011K::loop()
                     meter.updateMeterAllValues(METER_ALL_VALUES_CURRENT_L3_A, float(PrivCommRxBuffer[110]+256*PrivCommRxBuffer[111])/10);
 
                     // meter power
+                    static float prev_energy_rel_raw;
+                    static float prev_energy_abs_raw;
+                    float energy_rel_raw = float(PrivCommRxBuffer[84]+256*PrivCommRxBuffer[85])/1000;
```

Comment:
get float function: float energy_rel_raw = getFloatRx(PrivCommRxBuffer, 84)

Reply:
I'll do that in a separate PR because there are more places where the same thing can be improved.